test_runner.py

```diff
@@ -29,11 +29,12 @@ class OverrideDefinitions:
 
     override_args: Sequence[Sequence[str]] = tuple(tuple(" "))
     test_descr: str = "default"
+    test_id: str = "default"
```

Comment:
This feels like a `test_name` rather than `test_id`? could we rename to `test_name`?

Reply:
Changed 👍